docs/platform-services/threat-intelligence-indicators.md

```diff
@@ -68,75 +73,12 @@ You can also add threat intelligence indicators using the API or a collector. Se
 1. In Sumo Logic, go to **Manage Data > Logs > Threat Intelligence**.
 1. Click **Add Indicators**. The dialog displays. <br/><img src={useBaseUrl('img/platform-services/threat-intelligence-add-indicators.png')} alt="Add threat intelligence indicators" style={{border: '1px solid gray'}} width="500" />
```

Comment:
Similarly, I'm not sure what will be rendering the markdown, but Github needs the `![alt text](image URL)` format. But if it's some other renderer then maybe this works.

Reply:
Rendering is done by Docusaurus. The above renders as:

<img width="526" alt="Screenshot 2024-01-23 at 9 43 48 AM" src="https://github.com/SumoLogic/sumologic-documentation/assets/127961815/019780b2-9f98-4642-9945-51652dfeddeb">